### ISSUE TITLE
do we need sudo?

### ISSUE THREAD

**Issue by jamesbeedy** [erik78se/charm-gitlab-runner#22]
I'm thinking these hooks run as root, is there a reason to use `sudo`?

https://github.com/erik78se/charm-gitlab-runner/blob/b0030376ea053880e8fc4fb76a00c21d65685084/src/hooks/install#L11

**Comment by erik78se** [comment]
No, this is completely right and its removed in the operator version as this charm is mostly here for reference. Good spot.
